packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -35,7 +35,19 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     uint256 public liquidReserves;
 
     // Reserves currently utilized due to L2-L1 transactions in flight.
-    uint256 public utilizedReserves;
+    int256 public utilizedReserves;
+
+    // Exponential decay exchange rate to accumulate fees to LPs over time.
+    uint256 public lpFeeRatePerSecond;
+
+    // Last timestamp that LP fees were updated.
+    uint256 public lastLpFeeUpdate;
+
+    // Cumulative undistributed LP fees. As fees accumulate, they are subtracted from this number.
+    uint256 public undistributedLpFees;
+
+    // How long, in seconds, it takes for transfers to move from L2 optimism to L1 Ethereum.
+    uint256 public optimismL2toL1Time = 1 weeks;
```

Comment:
Should we allow this variable to be modified by an owner?

Reply:
I actually realized we don't need this variable anymore! It was when lingering from a previous implementation that used it to compute the exchange rate such that all fees are paid out over the duration of the L12->L1 bidding period; now, all this information is baked into `lpFeeRatePerSecond`